project3/styles.css

```diff
@@ -0,0 +1,321 @@
+*,
+*::before,
+*::after {
+    box-sizing: border-box;
+}
+
+body {
```

Comment:
Remember to also include base rules for `font-family` and text `color` on your `body` element.